packages/pn-commons/src/services/date.service.ts

```diff
@@ -11,7 +13,28 @@ export function formatDate(dateString: string): string {
   const date = new Date(dateString);
   const month = `0${date.getMonth() + 1}`.slice(-2);
   const day = `0${date.getDate()}`.slice(-2);
-  return isToday(date) ? 'Oggi' : `${day}/${month}/${date.getFullYear()}`;
+  const todayLabel = getLocalizedOrDefaultLabel(
+    'common',
+    'date-time.today-uppercase-initial',
+    "Oggi"
+  );
+  return isToday(date) ? todayLabel : `${day}/${month}/${date.getFullYear()}`;
+}
+
+export function formatTimeHHMM(dateString: string): string {
```

Comment:
Add tests for new functions

Reply:
Added, please note that I had to localize the hour in order to build the expected result; cfr. the comments in the test.